### ISSUE TITLE
Can not compile to kmodel

### ISSUE THREAD

**Issue by ligun** [kendryte/nncase#43]
I tried to compile the model from tflite to kmodel on Google Colaboratory..

```
./ncc/ncc compile /content/model.tflite /content/model.kmodel -i tflite -o kmodel --dataset /content/test/
```

But ncc output the error.
```
Fatal: Shapes must be same, but got [1x1x1x1] and [1x1x1x7]
```

How can I compile successfully?

**Comment by RobertoIV** [comment]
I'm having the same error, I have tried with the master branch and the alpha, but still can't be able to fix the issue.

**Comment by sunnycase** [comment]
Can you provide the tflite?

**Comment by RobertoIV** [comment]
You can check the following link for the tflite and the pb files:
https://drive.google.com/open?id=1yacOVg83nlf9Zu-_pdP4ih-XRhLDsEHM

Thank you

**Comment by sunnycase** [comment]
@RobertoIV  Fixed in 0e90433, you can try the newest ncc in the CI.

**Comment by ligun** [comment]
Thanks for your fix.
I tried to build and run master commit, and can get the kmodel file.
